linera-service/tests/util_tests.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#![cfg(feature = "rocksdb")]
+
+use linera_service::util;
+use std::{env, path::Path};
+
+#[test_log::test(tokio::test)]
+async fn test_resolve_binary_with_test_default() {
+    util::resolve_binary("linera", "linera-service")
+        .await
+        .unwrap();
+}
+
+#[test_log::test(tokio::test)]
+async fn test_resolve_binary_from_relative_path() {
+    util::resolve_binary_in_same_directory_as(
+        "../target/debug/xyz",
+        "linera-proxy",
+        "linera-service",
+    )
+    .await
+    .unwrap();
+}
+
+#[test_log::test(tokio::test)]
+async fn test_resolve_binary_from_absolute_path() {
+    let path = Path::new("../target/debug/linera").canonicalize().unwrap();
+    util::resolve_binary_in_same_directory_as(&path, "linera-proxy", "linera-service")
+        .await
+        .unwrap();
+}
+
+#[test_log::test(tokio::test)]
+async fn test_resolve_binary_from_test_path() {
+    util::resolve_binary_in_same_directory_as(
+        "../target/debug/deps/xyz",
+        "linera-proxy",
+        "linera-service",
+    )
+    .await
+    .unwrap();
+}
+
+#[test_log::test(tokio::test)]
+async fn test_resolve_binary_from_installed_path() {
+    let cwd = env::current_dir().unwrap();
+    let path_value = format!(
+        "{}/../target/debug:{}",
+        cwd.display(),
+        env::var("PATH").unwrap_or_default()
+    );
+    env::set_var("PATH", path_value);
+    util::resolve_binary_in_same_directory_as("linera", "linera-proxy", "linera-service")
+        .await
+        .unwrap();
```

Comment:
Should we assert that the returned paths are as expected?

Reply:
sure!